oidc/transport.go

```diff
@@ -47,24 +47,24 @@ type AuthenticatedTransport struct {
 	http.RoundTripper
 
 	mu  sync.Mutex
```

Comment:
Right. The AuthenticatedTransport takes a TokenRefresher which you can
customize to fit your needs.
On May 3, 2016 5:06 PM, "bobbyrullo" <notifications@github.com> wrote:

> In oidc/transport.go
> <https://github.com/coreos/go-oidc/pull/81#discussion_r61976618>:
>
> > @@ -47,24 +47,24 @@ type AuthenticatedTransport struct {
> >  	http.RoundTripper
> >
> >  	mu  sync.Mutex
>
> Note that this is for AuthenticatedTransport not TokenRefresher.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/go-oidc/pull/81/files/1b333838672f14dc4778d180ca3e1d0812fdea2a#r61976618>
>